frontend/packages/knative-plugin/src/types.ts

```diff
@@ -85,6 +85,18 @@ export type EventBrokerKind = {
   };
 } & K8sResourceKind;
 
+export enum TriggerConditionTypes {
+  Ready = 'Ready',
+  brokerReady = 'BrokerReady',
```

Comment:
any reason for having diff case here

Reply:
This is used in `TriggerRow.tsx`, but I can remove it and use string directly if it isn't required.